contrib/updates/Dockerfile.cosmovisor

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.18-alpine3.17 AS builder
```

Comment:
We dont really need the "AS builder" alias here since its not really a multi-stage build. :)